### DISCUSSION TITLE
Should the verbs of 46-text-1 be in present tense or past tense?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1132] [Q&A] [answered]
Currently, the `en` translations of 46-text-1 show verbs in present tense. I'm wondering if the correct tense should be present tense or past tense.

https://github.com/rdrg109/subtitles-npcr/blob/620c6f83099383a9224f17690686751975af2e08/sentences/46-text-2.yaml#L133-L136

If the translation above were in **past tense**, it would be: "so he **changed** his mind"

https://github.com/rdrg109/subtitles-npcr/blob/620c6f83099383a9224f17690686751975af2e08/sentences/46-text-2.yaml#L143-L146

If the translation above were in **past tense**, it would be: "even I **didn't want** to let her go."

https://github.com/rdrg109/subtitles-npcr/blob/620c6f83099383a9224f17690686751975af2e08/sentences/46-text-2.yaml#L148-L151

If the translation above were in **past tense**, it would be: "my daughter **saw** that everyone did not agree"

https://github.com/rdrg109/subtitles-npcr/blob/620c6f83099383a9224f17690686751975af2e08/sentences/46-text-2.yaml#L153-L156

If the translation above were in **past tense**, it would be: "and anxiously **cried**"

What do you think? Should the translation be shown in present or past tense?

**Comment by rodrigo-morales-1** [comment]
To me, the verbs should be shown in past tense. When we see the video, we might feel inclined to show verbs in present tense because we can see things happening as we see it. However, the speaker is the mom which is describing an experience  from the past.

@fhxshrs dp you agree with me?

**Reply by fhxshrs** [reply, marked as answer]
@rdrg109 I agree, the verbs should be shown in past tense.

**Reply by rodrigo-morales-1** [reply]
@fhxshrs Ok, thanks for the reply! ec84d0c
